src/integrations/admin/social-templates-integration.php

```diff
@@ -156,18 +155,20 @@ protected function build_social_fields( Yoast_Form $yform, $identifier, $page_ty
 		$image_url_field_id = 'social-image-url-' . $identifier;
 		$image_id_field_id  = 'social-image-id-' . $identifier;
 		$is_premium         = YoastSEO()->helpers->product->is_premium();
+		$is_premium_16_5    = defined( '\WPSEO_PREMIUM_VERSION' ) && \version_compare( \WPSEO_PREMIUM_VERSION, '16.5-RC0', '>=' );
```

Comment:
I think this name is a bit misleading. Not only 16.5 is fine, but everything above that as well.
Suggestion: `is_premium_16_5_or_up`. (but I'm open to other suggestions. I was also thinking about something like `is_correct_premium_version`)